op-batcher/flags/throttle_flags.go

```diff
@@ -0,0 +1,179 @@
+package flags
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	// Block-builder side
+	DefaultThrottleTxSizeLowerLimit    = 150
+	DefaultThrottleTxSizeUpperLimit    = 10_000
+	DefaultThrottleBlockSizeLowerLimit = 21_000
```

Comment:
I think this should be lower, this would still be 10.5kB/s with the strongest throttling intensity. Maybe 1000? Just thinking of the case where 50 rollups would max-throttle, then we'd still have 50x500kB/s=25kB/s with this setting while the max blob throughput is 96kB/s, and there would be a large backlog to clear first.

Reply:
I think you meant to say 50*10KB/s = 500KB/s. If max blob throughput is about 100 kB/s then we need to reduce by a factor of 5?

Working it back the other way, if we set the lower limit to 4000 bytes, then this would imply a throughput of 2KB/s, allowing for 96/2 = 48 rollups to operate at this level simultaneously. This is roughly the throughput we see on OPM.

Reply:
I went with 4000 bytes for now [3d6f521](https://github.com/ethereum-optimism/optimism/pull/17032/commits/3d6f5217b5aa4ecec04c942b20ea8c34cbfa5601)

Reply:
Yeah that works [edf8119](https://github.com/ethereum-optimism/optimism/pull/17032/commits/edf811969c350d14f4f2d75c88b56eb2aad83c43)